lib/surface/translator/tag_translator.ex

```diff
@@ -200,4 +201,8 @@ defmodule Surface.Translator.TagTranslator do
   defp replace_attribute_expr(value) do
     value
   end
+
+  defp value_is_nil?(value) do
+    is_list(value) and value |> Enum.filter(&(&1 !== 32)) |> Enum.empty?
```

Comment:
What about moving the `is_list` to a guard? I understand it would required adding another clause to the function, but it makes it a bit more clear what's happening, don't you think?

Also, if we are going to filter the empty spaces, what about moving the code 32 to a module variable like `@whitespace 32`, so it is clearer what this is doing?

Reply:
In terms of the feature itself I'll leave it up to @msaraiva, but I'm not sure we need to support removing empty strings as well. Do you have a specific use case in mind where it would be useful?

Reply:
> What about moving the is_list to a guard? I understand it would required adding another clause to the function, but it makes it a bit more clear what's happening, don't you think?

From this implementation, `value_is_nil?` needs to return a boolean, so adding a guard to the function would not pattern match on non list types, which would then need to be handled in a separate function. Is this what you are suggesting? I do agree this function is very ugly, and could prob be cleaned up. 

> Also, if we are going to filter the empty spaces, what about moving the code 32 to a module variable like @whitespace 32, so it is clearer what this is doing?

Yeah, thats prob a good idea for readability purposes. I'll let @msaraiva chime in on whether he thinks it is worth having this or not, as I agree it may not be worth it.

Reply:
Updated, the code just a bit. I implemented your whitespace idea, but just made it a local function variable instead of a module variable, seemed like the better route.

Reply:
> Is this what you are suggesting?

Yes. Having two clauses. But with @msaraiva's suggested changes there's no need for this.